src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/AnalyzeContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public class AnalyzeContext
```

Comment:
May be good to put an obsolete tag on as a warning

Reply:
Added